core/runtime/binaryen/wasm_memory_impl.hpp

```diff
@@ -24,6 +24,25 @@ namespace kagome::runtime::binaryen {
   // Alignment for pointers, same with substrate:
   // https://github.com/paritytech/substrate/blob/743981a083f244a090b40ccfb5ce902199b55334/primitives/allocator/src/freeing_bump.rs#L56
   inline const uint8_t kAlignment = sizeof(size_t);
+  inline const size_t kInitialMemorySize = 2 * 1u << 20;  // 2Mb
```

Comment:
Hm, we could make user-defined literals for Mb and Kb...

Reply:
Good idea. Thanks.